Frameworks/include/UIApplicationInternal.h

```diff
@@ -31,7 +31,7 @@
 - (void)_sendResumeEvent;
 - (void)_sendToastActionReceivedEvent:(NSDictionary*)toastAction;
 - (void)_sendVoiceCommandReceivedEvent:(WMSSpeechRecognitionResult*)result;
-- (void)_sendProtocolReceivedEvent:(WFUri*)uri;
+- (void)_sendProtocolReceivedEvent:(WFUri*)protocolUri source:(NSString*)source;
```

Comment:
yes, with a warning

---
In reply to: [95008850](https://github.com/Microsoft/WinObjC/pull/1643#discussion_r95008850) [](ancestors = 95008850)